types/jsrsasign/modules/KJUR/jws/JWS.d.ts

```diff
@@ -234,7 +234,7 @@ declare namespace jsrsasign.KJUR.jws {
          */
         function verify(
             sJWS: string,
-            key: string,
+            key: string | RSAKey | crypto.DSA | crypto.ECDSA,
```

Comment:
I think this is right, but do you have the docs for it?

https://kjur.github.io/jsrsasign/api/symbols/KJUR.jws.JWS.html#.verify seems vague about how keys exactly are supported here.